server/adapters/prismic.js

```diff
@@ -266,54 +266,68 @@ const getNavi = function ( type ) {
  * Load data for API response. Resolve RAW from Service.
  *
  */
-const getDataForApi = function ( req, listener ) {
-    return new Promise(( resolve, reject ) => {
-        const doQuery = function ( type ) {
-            prismic.api( core.config.api.access, null ).then(( api ) => {
-                const done = function ( json ) {
-                    resolve( json.results );
+const getDataForApi = function (req, listener) {
+    return new Promise((resolve, reject) => {
+        const doQuery = function (type) {
+            prismic.api(core.config.api.access, null).then((api) => {
+                const done = function (json) {
+                    resolve(json.results);
                 };
-                const fail = function ( error ) {
+                const fail = function (error) {
                     resolve({
                         error: error
                     });
                 };
-                const form = getForm( req, api, type );
+                // const form = getForm(req, api, type);
                 let query = [];
 
+                let newType = type;
+
+                if (type === "home") {
+                    newType = "project";
+                } else if (type === "studio") {
+                    newType = "page";
+                } else if (type === "work") {
+                    newType = "project";
+                } else if (type === "play") {
+                    newType = "blog";
+                }
```

Comment:
These are the core changes since the `api.form` api collections are deprecated we cannot query by those anymore. In order for things to work we have to switch to `api.query` and reverse map the collection types back to raw document types.